### PR TITLE
add 290. Word Pattern

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 20m
Please check my code.

### CODE REVIEW

File: 290_word-pattern.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        s_list = s.split()
+        if len(pattern) != len(s_list):
+            return False
+
+        for alph, word in zip(pattern, s_list):
+            if alph in ref:
+                if word != ref[alph]:
+                    return False
+
+            elif word in ref.values():
```
Comment (ryuji0123) [inline 290_word-pattern.py:24]: O(len(ref))

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
solve this with O(len(pattern))

**Comment by kobakobashu** [comment]
I solved this with O(len(pattern)).  Please check my code.

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request closed without merging**